apollo-federation/src/query_plan/query_planner.rs

```diff
@@ -196,6 +195,19 @@ impl QueryPlannerConfig {
     }
 }
 
+#[derive(Debug, Default, Clone)]
+pub struct QueryPlanOptions {
+    /**
+     * A set of labels which will be used _during query planning_ to
+     * enable/disable edges with a matching label in their override condition.
+     * Edges with override conditions require their label to be present or absent
+     * from this set in order to be traversable. These labels enable the
+     * progressive @override feature.
+     */
```

Comment:
```suggestion
    /// A set of labels which will be used _during query planning_ to
    /// enable/disable edges with a matching label in their override condition.
    /// Edges with override conditions require their label to be present or absent
    /// from this set in order to be traversable. These labels enable the
    /// progressive @override feature.
```
doc comment syntax 😇